### DISCUSSION TITLE
Few questions

### DISCUSSION THREAD

**Discussion by markob94** [SpiralLinux/SpiralLinux-project#329] [General]
Hi,

I just found this project. It looks really good , especially how "hard" is to install debian and implement Snapper and snapshots.

I saw on main page that Gnome is not on latest version . Is it possible to manually update it ? 

Also in case we want completely switch to Debian ,stable or testing( as we install Debian iso), that is possible ? 

Thanks,

**Comment by geckolinux** [comment]
Hi there, thanks for the questions.

> I saw on main page that Gnome is not on latest version . Is it possible to manually update it ?

After installing SpiralLinux if you stay on Debian Stable then you must use the package versions that are available in Debian Stable and optionally in Debian Backports. Debian will never release newer Gnome versions for Debian 12 during its supported lifecycle. First it would be good to ask yourself if you really need a newer Gnome version, as the version in Debian 12 with the extensions and additional polish that SpiralLinux offers makes Gnome a lot more usable in my opinion. But if you really need/want the latest Gnome release, then Debian Stable is definitely not the right OS for you. You could consider GeckoLinux _Rolling_, which shares the same concepts as SpiralLinux except it's built from openSUSE Tumbleweed, which offers the latest package versions with fairly good stability.

> Also in case we want completely switch to Debian ,stable or testing( as we install Debian iso), that is possible ?

Sorry, I didn't understand this question. SpiralLinux uses Debian Stable, with optional packages from Backports. After installation you can switch your system to Debian Testing or Unstable:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#switching-from-debian-stable-to-the-testing-or-unstable-branch
